frontend/src/lib/k8s/cluster.ts

```diff
@@ -686,18 +686,19 @@ export function makeKubeObject<T extends KubeObjectInterface | KubeEvent>(
 
       type ApiEndpointWithScale = {
         scale: {
-          put: (
-            data: { metadata: KubeMetadata; spec: { replicas: number } },
```

Comment:
Hmm, I just noticed that this is part of the lib which may be used by plugins. So you should keep the put but just make it do the same thing as patch.